src/components/Multiselect/Multiselect.vue

```diff
@@ -20,20 +20,20 @@
         <Badge
           v-for="option in modelValue"
           :key="option.label || option"
-          variant="turquoise"
           :size="size"
-          class="my-2 ml-2"
         >
           <div class="flex items-center">
             {{ option.label || option }}
-            <LobButton
-              variant="none"
-              class="mt-0.5 ml-1 h-min"
+            <button
+              class="ml-2 -mr-1 h-min"
               @click="() => handleOptionDeselect(option)"
             >
-              <XmarkLarge size="s" />
+              <XmarkLarge
+                size="s"
+                class="h-3"
+              />
               <span class="sr-only">{{ t('multiselect.deselectLabel') }}</span>
```

Comment:
nit: if the only purpose of this `<span>` is to provide text that will be read by a screen reader then this could instead be supplied as an `aria-label` on the `button` element.